java/gazelle/private/java/java.go

```diff
@@ -21,7 +21,7 @@ func IsTestPackage(pkg string) bool {
 		}
 	}
 
-	return strings.Contains(pkg, "/test/")
+	return strings.Contains(pkg, "/test")
```

Comment:
This would cause issues for packages with names like `testutils`, but probably a trailing `test` package segment makes sense to check for - WDYT?

```suggestion
	return strings.Contains(pkg, "/test/") || strings.HasSuffix(pkg, "/test")
```

Reply:
You're totally right! Thanks!